transactron/utils/amaranth_ext/shifter.py

```diff
@@ -287,7 +287,7 @@ def generic_shift_vec_left(
     Sequence[Value | ValueCastable]
         The shifted sequence, the same length as `data1`.
     """
-    return list(reversed(generic_shift_vec_right(list(reversed(data1)), list(reversed(data2)), offset)))
+    return list(reversed(generic_shift_vec_right(list(reversed(data1)), list(reversed(data2)), offset)))  # type: ignore
```

Comment:
Remove ValueCastable from data[1|2] method and in docstring, the same in shift_vec_right docstring